artemis-server/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/NettyAcceptor.java

```diff
@@ -729,6 +731,7 @@ public NettyServerConnection createConnection(final ChannelHandlerContext ctx,
                   public void operationComplete(final io.netty.util.concurrent.Future<Channel> future) throws Exception {
                      if (future.isSuccess()) {
                         active = true;
+                        ctx.channel().pipeline().remove("readTimeoutHandler");
```

Comment:
This is only valid for SSL connections.  The "readTimeoutHandler" should be removed for non-SSL connections as well.